net/goai/goai_shema.go

```diff
@@ -204,6 +205,61 @@ func (oai *OpenApiV3) structToSchema(object interface{}) (*Schema, error) {
 			if validationRuleSet.Contains(validationRuleKeyForRequired) {
 				schema.Required = append(schema.Required, key)
 			}
+
+			// Extract validation rules to schema. like min, max, length
+			validationRules := ref.Value.ValidationRules
+			// remove custom error message
+			if idx := strings.Index(ref.Value.ValidationRules, "#"); idx != -1 {
+				validationRules = ref.Value.ValidationRules[:idx]
+			}
+			lstRules := gstr.Split(validationRules, "|")
+			for _, rule := range lstRules {
+				if strings.HasPrefix(rule, "max") {
```

Comment:
Do not use hard-coded string, use constant instead. Please refer current usage example: https://github.com/gogf/gf/blob/7d4c59ac5aea64a6db83e79d6bb7aee1ed66250e/net/goai/goai.go#L66

Reply:
Indeed, hard-coded is not good practice.